protocol/daemons/pricefeed/client/price_function/binance/exchange_query_details.go

```diff
@@ -6,17 +6,16 @@ import (
 )
 
 var (
-	// Example request URL: /api/v3/ticker/24hr?symbols=["BTCUSDT","BNBUSDT"]
 	BinanceDetails = types.ExchangeQueryDetails{
 		Exchange:      exchange_common.EXCHANGE_ID_BINANCE,
-		Url:           "https://data.binance.com/api/v3/ticker/24hr?symbols=[$]",
+		Url:           "https://data.binance.com/api/v3/ticker/24hr",
```

Comment:
I believe the number of requests did not change, because these were already multimarket exchanges, meaning that we periodically make 1 request for a number of tickers. However, binance does use request weights - so I think our increased from 33-40 per request. However, I'm not seeing 429s when running locally on binance, so it looks like we are ok.